actions/lint-insights/action.yaml

```
@@ -0,0 +1,36 @@
+name: 'Lint Security Insights File'
+description: 'Use cue to validate a given security insights file against the SI schema.'
+
+inputs:
+  si_path:
+    description: 'Relative security-insights.yml path'
+    required: false
+    default: 'security-insights.yml'
+  schema_version:
+    description: 'Version of the SI schema to use'
+    required: false
+    default: 'v2.1.0'
+  cue_setup:
+    description: 'Run setup-cue (defaults to true)'
+    required: false
+    default: true
```

https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault says this is a string, so let's quote it.

```suggestion
    default: "true"
```